### ISSUE TITLE
Expired Bounty not available to use.

### ISSUE THREAD

**Issue by Codes4Fun** [bountysource/core#1028]
I have made two bounties in the past, both of which were set to expire in three months and have since gone past that expiration time. One of them was closed as the developer wasn't going to add the feature, and the other had no responses at all.

It's not clear how I am supposed to reuse my bounties funds or how to get them returned?

**Am I supposed to claim them for myself?** If so, maybe it would help if this were given a proper interface or was documented somewhere (last I checked I couldn't find an answer to this issue).

**Comment by rappo** [comment]
@Codes4Fun currently this is an admin / support task. Please email support@bountysource.com with your details. I've renamed this issue and will leave it open, since we do need a proper way for end users to self-serve on the front end.

**Comment by teran-mckinney** [comment]
I have this same problem. I've emailed support@bountysource.com twice and have not heard back about refunding my expired bounty.

**Comment by rappo** [comment]
Adding details because I'm going to put a bounty on this. My criteria for closing this issue is:

- [ ] Add a Refund button to issues that have not been paid out on https://www.bountysource.com/activity/bounties
- [ ] Upon clicking Refund, go to a confirmation page that provides detail on the amount, the issue, and where the funds will go (personal account or team account)
- [ ] Refund transactions should mirror what the "Refund Bounty" button does on admin/issues/
- [ ] Refunds should only be allowed if there's no active developer on the issue -- we may need to decide what this means
- [ ] If the criteria for the bullet point above is met, show a refund button on the issue page (only to the backer, only if logged in...) that goes to the above confirmation page.

**Comment by ldrrp** [comment]
if someone has a large bounty they should put part of that bounty as a bounty for this so that when its completed you get part of it back at least haha.

**Comment by tuxayo** [comment]
I though there wasn't a bounty for this but actually there is!
![current bounty](https://api.bountysource.com/badge/issue?issue_id=34504010)
https://www.bountysource.com/issues/34504010-add-a-refund-bounty-option-to-the-front-end

It would be a great addition. I'm a bit reluctant to use Bountysource currently because I won't be able to **easily** reorganize my bounties every few months. To avoid having a lot of money kind of trapped on issues that will never be implemented.
Given a fixed amount of money that one wants to have sitting in Bountysource, or a fixed monthly/yearly budget. Such ease of reorganization could allow one to have a much of relevant use of it's own budget.

It can also reassure newcomers who could fear getting scammed seeing that there is no easy way to get a bounty refunded. Anything that helps building trust will help this great economic model to spread :D

**Comment by msm1089** [comment]
I am looking into adding this feature, but have not been able to get the site running in docker on windows. I notice that the instructions for Local Development Environment with Docker are deprecated, but tried them to see how far I got. All the commands run, but it seems the bountysource and sphinx containers cannot start.

**For bountysource I get "/app/Gemfile not found".
For Sphinx I get "FATAL: no readable config file (looked in /usr/local/etc/sphinx.conf, ./sphinx.conf)."**

Is anyone able to provide working instructions for Docker for Windows? Would really like to work on this, thanks in advance :)

**Comment by LemonAndroid** [comment]
Oh actually it is more complicated then that. Or are you planning on collecting a 25$ bountysource fee (completely intransparent) as well for refunds?

https://gyazo.com/cce8c4a46dd8af42085fd46f1393fd04


Good luck with CanYa going downhill. I'm out.

**Comment by LemonAndroid** [comment]
https://gyazo.com/9a81ef3c2e40b920239b053bc23d341a?token=63f04cbb498b9bdbc1b164ec490aaa4a

That's basically a lie, it's 260$ - 10%(260$) => 236$. What a rip off.

**Comment by LemonAndroid** [comment]
It's not ok to advertise a bounty with it's theoretical worth.

**Comment by tuxayo** [comment]
> That's basically a lie, it's 260$ - 10%(260$) => 236$. What a rip off.

Indeed that's not great.  (and that's not even enough to make them sustainable apparently) A way to improve that would be that more users of Bountysource would donate to them directly:
https://salt.bountysource.com/teams/bountysource

Even better: **additionally** Bountysource would fix thresholds of donations at which the cut would be decreased down to only financial fees. That seem to me a better funding model. More transparent, and less scary for potential donator to the projects.

> It's not ok to advertise a bounty with it's theoretical worth.

So hopefully such a difference will disappear someday.

**Comment by seanlynch** [comment]
Wow, it's pretty horrendous that there is no frontend to deal with expired bounties. I won't be using Bountysource anymore until this is fixed.

**Comment by tuxayo** [comment]
Or maybe use it and manually ask to cancel bounties. Having a lot of demand for cancellation will create an instigation to have it automated.

**Comment by seanlynch** [comment]
The problem is that having the expiration on bounties out of sight and out of mind smacks of dishonesty on BS's part. At the very least, they haven't prioritized implementing it despite forcing people to pledge a minimum amount to even have an expiration in the first place.

**Comment by theoryshaw** [comment]
I would agree with @seanlynch.  I'm hesitate to use this platform, since the reimbursement mechanism is not front and center, and user friendly.  Smacks of shady.
Also, there's the sense out there, with a few google searches, that the reimbursement process through BS is slow and convoluted.

**Comment by knocte** [comment]
Agree with @theoryshaw, that's why these days I myself am using gitcoin. If this was implemented, I would switch to bountysource... I'd love to contribute USD to this bounty (https://www.bountysource.com/issues/34504010-add-a-refund-bounty-option-to-the-front-end) but I can only use cryptocurrency, and it seems adding crypto to an already-started-as-fiat bounty is not possible. If anyone here can contribute non-crypto, please do so to attract some dev that tackles this.

**Comment by tuxayo** [comment]
> I can only use cryptocurrency, and it seems adding crypto to an already-started-as-fiat bounty is not possible.

Is there an issue for that where you could create a crypto-bounty? :P

> If anyone here can contribute non-crypto, please do so to attract some dev that tackles this.

Thanks, you motivated me to throw some more money in here.

**Comment by davidbaumann** [comment]
Hello, so what is the status here?
